qabel_index/settings.py

```diff
@@ -124,12 +139,22 @@
 
 # Application configuration
 
+PROMETHEUS_EXPORT_MIGRATIONS = False
```

Comment:
nb. setting added in django-prometheus fork. This works around a very-early uninterceptible databaseaccess if this is left enabled, which would hit a database that is not yet up.

For us this metric is irrelevant.